client/src/js/views/cloudupload.js

```diff
@@ -0,0 +1,71 @@
+define([
+    'backbone',
+    'views/dialog',
+    'templates/dc/cloudupload.html',
+    'jquery.cookie'
+], function(
+    Backbone,
+    DialogView,
+    template
+) {
+
+    return DialogView.extend({
+        template: template,
+        title: 'Upload to CCP4 Cloud',
+
+        buttons: {
+            'Upload': 'upload',
+            'Cancel': 'closeDialog',
+        },
+        
+        ui: {
+            username: 'input[name=username]',
+            cloudrunid: 'input[name=cloudrunid]',
+            remember: 'input[name=remember]',
+        },
+        
+        upload: function() {
+            if (this.ui.remember.is(':checked')) {
+                let cookieOpts = { expires: 365, path: '/' }
+                $.cookie('ccp4_username', this.ui.username.val(), cookieOpts)
```

Comment:
Could we use the native cookie management library instead, since jQuery Cookie was removed? I'm happy to do it 

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/cookies/set
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/cookies/get